roles/pulp_repository/tasks/container.yml

```diff
@@ -8,6 +8,8 @@
     name: "{{ item.name }}"
     state: "{{ item.state }}"
   with_items: "{{ pulp_repository_container_repos }}"
+  loop_control:
+    label: "{{ item.name }}"
 
```

Comment:
Do you also need `no_log: true` set on the task?

Reply:
In a normal run, Ansible won't show the items with this label applied - each item will just show the name. If you use verbose mode, it will show the items.